xonsh/tools.py

```diff
@@ -1787,12 +1809,12 @@ def to_dict(x):
     return x
 
 
-def to_str_str_dict(x):
+def to_tok_color_dict(x):
     """Converts a string to str:str dictionary"""
-    if is_str_str_dict(x):
+    if is_tok_color_dict(x):
         return x
     x = to_dict(x)
-    if not is_str_str_dict(x):
+    if not is_tok_color_dict(x):
         msg = f'"{x}" can not be converted to str:str dictionary.'
         warnings.warn(msg, RuntimeWarning)
         x = dict()
```

Comment:
I think the docstring and the warning message here should be updated to reflect that it's a `Token: str` dictionary